Assets/Shaders/PhysicsDebugPixel.frag

```diff
@@ -1,11 +1,24 @@
-#version 450 core
-#extension GL_ARB_separate_shader_objects : enable
-
-layout(location = 0) in vec4 fragColor;
-
-layout(location = 0) out vec4 finalColor;
-
-void main()
-{
-    finalColor = vec4(fragColor.xyz, 1.0f);
+<<<<<<< HEAD
```

Comment:
It seems to be some unresolved conflicts in this file. The strings inserted by git when conflicts occur when merging are still in this file. Like the "<<<<<< HEAD", "=====" and ">>>>>> master".

The same problem is in the file PhysicsDebugPixel.vert